vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -392,9 +392,18 @@ void ReactInstanceWin::Initialize() noexcept {
 #endif
         case JSIEngine::V8:
 #if defined(USE_V8)
-          preparedScriptStore =
-              std::make_unique<facebook::react::BasePreparedScriptStoreImpl>(getApplicationLocalFolder());
-
+        {
+          uint32_t length{0};
+          if (GetCurrentPackageFullName(&length, nullptr) != APPMODEL_ERROR_NO_PACKAGE) {
+            preparedScriptStore =
+                std::make_unique<facebook::react::BasePreparedScriptStoreImpl>(getApplicationTempFolder());
+          } else {
+            char tempPath[MAX_PATH];
+            if (GetTempPathA(MAX_PATH, tempPath)) {
```

Comment:
```suggestion
            if (GetTempPathW(MAX_PATH, tempPath)) {
```

Should this be the unicode-aware version? IIRC we've had this snippet of code copied around for a while...